### ISSUE TITLE
rpc error: code = Unknown desc = no such table: canary_domains

### ISSUE THREAD

**Issue by hippich** [BishopFox/sliver#316]
**Describe the bug**
When generating an implant, specifying canary domain throws an error

```
rpc error: code = Unknown desc = no such table: canary_domains
```

**To Reproduce**

1. Launch v1.2 of `./sliver-server`
2. Command: `generate --dns v.domain.com --canary c.domain.com -e -r service`
3. See error `rpc error: code = Unknown desc = no such table: canary_domains`

```
sliver > generate --dns v.domain.com. --canary c.domain.com. -e -r service

[*] Generating new windows/amd64 implant binary
[*] Symbol obfuscation is enabled.
[*] This process can take awhile, and consumes significant amounts of CPU/Memory
[!] rpc error: code = Unknown desc = no such table: canary_domains

sliver > generate --dns v.domain.com. -e -r service

[*] Generating new windows/amd64 implant binary
[*] Symbol obfuscation is enabled.
[*] This process can take awhile, and consumes significant amounts of CPU/Memory
[*] Build completed in 00:29:35
[*] Implant saved to /root/COMFORTABLE_MATTER.exe
```

**Expected behavior**
No error

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04 (server)

**Comment by moloch--** [comment]
Oh this one is probably my fault in the move to SQL.

**Comment by moloch--** [comment]
@hippich should be fixed in [v1.2.1](https://github.com/BishopFox/sliver/releases/tag/v1.2.1)
